pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -182,7 +185,15 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 				Protocol:      v1.ProtocolTCP,
 			},
 		},
-		Env:       opspec.DaemonEnvVars(c.cephVersion.Image),
+		Env: append(
+			opspec.DaemonEnvVars(c.cephVersion.Image),
+			[]v1.EnvVar{
+				{Name: "ROOK_SYSTEM_NS", Value: operatorNamespace},
+				{Name: "ROOK_API_VERSION", Value: rookv1.Version},
+				{Name: "ROOK_VERSION", Value: c.rookVersion},
```

Comment:
`c.rookVersion` will be the image name rather than a raw version number. Try using [this version](https://github.com/rook/rook/blob/master/pkg/version/version.go#L19), which is updated by the build process.